Configurations/windows-makefile.tmpl

```diff
@@ -412,7 +412,7 @@ install_dev:
 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "$(SRCDIR)\ms\applink.c" \
 				       "$(INSTALLTOP)\include\openssl"
 	@rem {- output_on() unless grep { $_ eq "OPENSSL_USE_APPLINK" } @{$config{defines}}; "" -}
-	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "-exclude_re=/__DECC_" \
+	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "-exclude_re=\\__DECC_" \
```

Comment:
But there is `$arg =~ s|\\|/|g;` so original should have worked. Either way *formally* speaking pattern is not right, because it means that user may not call subdirectory `__DECC_`. I mean expression should be .h on the end and no delimiters between it and `__DECC_`